src/v3/src/types/schema.ts

```diff
@@ -400,6 +400,7 @@ export interface InfoboxElement extends UISchemaElement {
     contentType: string;
     title?: string;
     dataSe?: string;
+    bullets?: string[];
```

Comment:
I think `contentType` was an incomplete way of trying to implement something like this but given its not used here, I think you can delete it. 

Additionally, I would rename `bullets` to something like `listOptions` and then make the type match the options from the ListElement like 

`listOptions?: ListElement['options']`